recipes-extended/baseboxd-tools/baseboxd-tools/bundle-debug-info

```diff
@@ -218,6 +265,9 @@ fi
 while [ $# -gt 0 ]; do
   _cli_argument=$1
   case $_cli_argument in
+    -i | --ignore-errors)
+      IGNORE_ERRORS=1
```

Comment:
The way this is implemented the "-i" option must come before any arguments that might cause an error. It's not a big deal because users are expected to stick with the default ("all"), but it's not pretty. Maybe we should change the usage text to list options (-h, -i) and arguments (pkg, log, net, etc.) separately and then indicate that options must come before arguments.

Reply:
I think it would be better to rework it in a way that the order does not matter instead of enforcing arbitrary orders.

Reply:
It is quite common for tools to require that options be given before additional arguments, so I wouldn't call that "arbitrary". But if you'd like to rewrite the code using getopts, I have no objections. Of course, you can also use an environment variable instead of an option for a global override; then, order doesn't matter, either.